### PR TITLE
fix the min_rancher_version on monitoring and gke-operator

### PR DESCRIPTION
**Pull request by jiaqiluo**

### CODE REVIEW

**Review by aiyengar2 — DISMISSED** [review]
LGTM, talked to @jiaqiluo offline and the reason for this change seems to be that the old semver package that Rancher uses does not properly validate semver in that `alpha` < `patch` evaluates false.

**Review by dramich — DISMISSED** [review]

**Comment by aiyengar2** [comment]
The reason why this is required is due to the custom ordering of semver used by the Rancher dependency in https://github.com/mcuadros/go-version/blob/master/compare.go#L13

File: charts/rancher-gke-operator/1.0.100/questions.yaml [hunk 1]
```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.8-alpha1
```
Comment (jiaqiluo) [inline charts/rancher-gke-operator/1.0.100/questions.yaml:1]: This is the version in the release-v2.5 branch today.

**Review by aiyengar2 — DISMISSED** [review]

File: charts/rancher-gke-operator/1.0.200/questions.yaml [hunk 2]
```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.8-alpha1
```
Comment (jiaqiluo) [inline charts/rancher-gke-operator/1.0.200/questions.yaml:1]: gke-operator 1.0.200 is the version that will be released in rancher v2.5.10 in future

File: charts/rancher-gke-operator/1.0.100/questions.yaml [hunk 3]
```diff
@@ -1 +1 @@
...
+rancher_min_version: 2.5.8-patch1
```
Comment (jiaqiluo) [inline charts/rancher-gke-operator/1.0.100/questions.yaml:1]: gke-operator 1.0.100 is the version in the release-v2.5 branch today.

File: charts/rancher-gke-operator/1.0.200/questions.yaml [hunk 4]
```diff
@@ -1 +1 @@
...
+rancher_min_version: 2.5.8-patch1
```
Comment (jiaqiluo) [inline charts/rancher-gke-operator/1.0.200/questions.yaml:1]: gke-operator 1.0.200 is the version that will be released in rancher v2.5.10 in future

File: charts/rancher-monitoring/v0.2.2/questions.yml [hunk 5]
```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.8-rc1
+rancher_min_version: 2.5.8-patch1
```
Comment (jiaqiluo) [inline charts/rancher-monitoring/v0.2.2/questions.yml:1]: monitoring 0.2.2 is the version in the release-v2.5 branch today.

**Review by dramich — APPROVED** [review]

**Review by aiyengar2 — APPROVED** [review]
LGTM

**Pull request merged**